job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/ScmContext.groovy

```diff
@@ -152,7 +152,7 @@ class ScmContext extends AbstractExtensibleContext {
     }
 
     /**
-     * Adds a Git SCM source.
+     * Adds a Git SCM source. The "Create a tag for every build" option will be enabled by default.
```

Comment:
Maybe add an explanation that behavior is for backward compatibility with the plugin 1.0